### ISSUE TITLE
Not possible to specify reactListKey properly on tables

### ISSUE THREAD

**Issue by andymantell** [surevine/govuk-react-jsx#113]
Due to the structure of the table data, it's currently not possible to specify reactListKey on each row. You can specify it on the cells, but not the rows.

Will need to restructure the data slightly to make this possible so will probably be a major release.

cc @AhdabNasir

As an aside to this, it would be a good idea to add examples for each component demonstrating the use of reactListKey - both for documentation, but also to ensure it actually works across all the components.

**Comment by andymantell** [comment]
@AhdabNasir I've published `govuk-react-jsx@6.0.0-rc1` which should fix these issues. Would you be able to give it a whirl and see if it sorts things for you?

You will need to explicitly install that version - i.e. `npm install govuk-react-jsx@6.0.0-rc1` as it is a pre-release.

Please also note that the table data structure has changed and now looks like this:

```

<Table
  head={[
    {
      children: 'Month you apply',
      reactListKey: 'your-stable-row-key-here-0'
    },
    {
      children: 'Rate for bicycles',
      format: 'numeric',
      reactListKey: 'your-stable-row-key-here-1'
    },
    {
      children: 'Rate for vehicles',
      format: 'numeric',
      reactListKey: 'your-stable-row-key-here-2'
    }
  ]}
  rows={[
    {
      cells: [
        {
          children: 'January',
          reactListKey: 'your-stable-cell-key-here-0'
        },
        {
          children: '£85',
          format: 'numeric',
          reactListKey: 'your-stable-cell-key-here-1'
        },
        {
          children: '£95',
          format: 'numeric',
          reactListKey: 'your-stable-cell-key-here-2'
        }
      ],
      reactListKey: 'your-stable-row-key-here-0'
    },
    {
      cells: [
        {
          children: 'February',
          reactListKey: 'your-stable-cell-key-here-0'
        },
        {
          children: '£75',
          format: 'numeric',
          reactListKey: 'your-stable-cell-key-here-1'
        },
        {
          children: '£55',
          format: 'numeric',
          reactListKey: 'your-stable-cell-key-here-2'
        }
      ],
      reactListKey: 'your-stable-row-key-here-1'
    },
    {
      cells: [
        {
          children: 'March',
          reactListKey: 'your-stable-cell-key-here-0'
        },
        {
          children: '£165',
          format: 'numeric',
          reactListKey: 'your-stable-cell-key-here-1'
        },
        {
          children: '£125',
          format: 'numeric',
          reactListKey: 'your-stable-cell-key-here-2'
        }
      ],
      reactListKey: 'your-stable-row-key-here-2'
    }
  ]}
/>
```

Note the addition of the `cells` key to each row - allowing me to also specify a reactListKey for each row. This functionality was previously missing.

Let me know how you get on and I will publish it properly if it works for you.

**Comment by AhdabNasir** [comment]
@andymantell Thank you, I've installed the version and tried it and it works great! I no longer get the key warning issue

**Comment by andymantell** [comment]
ah fantastic, thanks for letting me know. I'll look at getting this released properly tomorrow.

**Comment by andymantell** [comment]
Hi @AhdabNasir

Really sorry this took so long, but I have now released govuk-react-jsx@6.0.0. Hopefully you've been ok on the pre-release version in the meantime.

Please note this version also includes an update to govuk-frontend@3.12.0

Hope this helps. Cheers.
